crates/oxc_linter/src/rules/react/style_prop_object.rs

```diff
@@ -153,7 +153,7 @@ impl Rule for StylePropObject {
             })
             .unwrap_or_default();
 
-        allow.sort();
+        allow.sort_unstable();
```

Comment:
`Ordering::Equal` is returned only when the value is same so we can use unstable sort here.